examples/main.go

```diff
@@ -35,7 +44,18 @@ func main() {
 			return
 		}
 		// Log raw message struct
-		fmt.Println(mes)
+		//fmt.Println(mes)
```

Comment:
yea... though I have some plans for maybe removing the entire bot.Incoming thing.. what is it needed for? If anyone wants all the messages they can just use a trigger